tools/spi-hdlc-adapter/spi-hdlc-adapter.c

```diff
@@ -154,9 +154,10 @@ static int sIntGpioValueFd = -1;
 static int sHdlcInputFd  = -1;
 static int sHdlcOutputFd = -1;
 
-static int     sSpiSpeed   = 1000000; // in Hz (default: 1MHz)
-static uint8_t sSpiMode    = 0;
-static int     sSpiCsDelay = 20; // in microseconds
+static int     sSpiSpeed      = 1000000; // in Hz (default: 1MHz)
+static uint8_t sSpiMode       = 0;
+static int     sSpiCsDelay    = 20; // in microseconds
+static int     sSpiResetDelay = 1;  // in milliseconds
```

Comment:
Suggest using zero as default the keep behavior same as before.

Reply:
Updated